README.md

```diff
@@ -2,7 +2,9 @@
 
 # Kubernetes controller-runtime Project
 
-The Kubernetes controller-runtime Project is a set of go libraries for building Controllers.
+The Kubernetes controller-runtime Project is a set of go libraries for building
```

Comment:
Maybe something more generic that clarifies that c-r can be used perfectly well without Kubebuilder, e.G.

`It is used by Kubebuilder and Controller-SDK. Those two are a great place to start for new projects. Check out this sample in Kubebuilder repo on how it can be used`

Reply:
Done.